crates/forge/tests/data/wasm_oracles/wasm_oracle/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[workspace]
+
+[package]
+name = "wasm_oracle"
```

Comment:
super tiny nit: maybe drop “wasm” in the name - it’s more about the format than what this module does. Something like `hello_oracles` or `simple_oracle` would fit better 😅